server/spec/refresh_pools_spec.rb

```diff
@@ -217,10 +214,14 @@
     product2 = create_product(name, name, :owner => owner2['key'])
 
     # migrate the subscription to another owner.
+    sub = get_hostedtest_subscription(pool.subscriptionId)
     sub["owner"] = owner2
-    @cp.update_subscription(sub)
-    @cp.list_subscriptions(owner1["key"]).length.should == 0
-    @cp.list_subscriptions(owner2["key"]).length.should == 1
+    update_hostedtest_subscription(sub)
+    #TODO - removing sleeps gives concurrent modification errrors - investigate
```

Comment:
Concurrent modification where? If it's in CP, I would argue it needs to be fixed rather than hiding it via sleeps -- actual clients won't be so kind.